cyvlfeat/sift/phow.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import division
+import math
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.image as mpimg
+import matplotlib.colors as colors
+import dsift
```

Comment:
Don't use relative importing that isn't really relative as this may break in unexpected ways. This import should look like `from . import dsift`